File: React-FrontEnd/VoyageRoyale-React/src/models/RentalModel/responses/getCustomRentalResponse.ts
export interface GetCustomRentalResponse{
	id: number;
	userId: number;
	createdDate:string;
	startDate: string;
	endDate: string;
	startKilometer: number;
	endKilometer: number;
	carDailyPrice: number;
	carPlate: string;
	userEmail: string;
	carImagePath: string;
	carBrandName: string;
	carModelName: string;
	carPositionCity: string;
	invoicesTotalPrice: string;
}

File: Java-Backend/VoyageRoyale-Java/src/main/java/com/tobeto/pair5/services/dtos/rental/responses/GetCustomRentalResponse.java
package com.tobeto.pair5.services.dtos.rental.responses;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class GetCustomRentalResponse {
    private int id;
    private LocalDate startDate;
    private LocalDate  endDate;
    private int startKilometer;
    private int endKilometer;
    private String carPlate;
    private String carImagePath;
    private String userEmail;
    private float invoicesTotalPrice;
    private float carDailyPrice;
}
